src/pages/instances/InstanceList.tsx

```diff
@@ -731,23 +736,26 @@ const InstanceList: FC = () => {
                       )
                     }
                   >
-                    <TableColumnsSelect
+                    <ColumnSelector
                       columns={USER_HIDEABLE_COLUMNS.filter((column) => {
                         if (column === CLUSTER_MEMBER && !isClustered) {
                           return false;
                         }
                         return true;
                       })}
-                      hidden={userHidden}
+                      userHidden={userHidden}
                       sizeHidden={sizeHidden}
-                      setHidden={setHidden}
+                      setUserHidden={setHidden}
                       className={classnames({
                         "u-hide": panelParams.instance,
                       })}
                     />
                     <SelectableMainTable
                       id="instances-table"
-                      headers={getHeaders(userHidden.concat(sizeHidden))}
+                      headers={visibleHeaderColumns(
+                        HEADERS,
+                        userHidden.concat(sizeHidden),
+                      )}
```

Comment:
```suggestion
                      headers={visibleHeaders}
```
I think we can use the variable from above here to simplify.

Reply:
They are not in the same scope. But I can move `visibleHeaders` so that it's accessible here too.